### ISSUE TITLE
Crashloopbackoff of wego-app using gitops install

### ISSUE THREAD

**Issue by mtparet** [weaveworks/weave-gitops#1177]
**Describe the bug**
Gitops install does not set the right tag. Manually replacing 0.5.0 by v0.5.0 works.
```
gitops install
kubectl describe pod wego-app-5c69b6985c-r9kd5 -n wego-system
```

```
 Events:
  Type     Reason     Age                 From               Message
  ----     ------     ----                ----               -------
  Normal   Scheduled  21m                 default-scheduler  Successfully assigned wego-system/wego-app-5c69b6985c-r9kd5 to minikube
  Normal   Pulling    20m (x4 over 21m)   kubelet            Pulling image "ghcr.io/weaveworks/wego-app:0.5.0"
  Warning  Failed     20m (x4 over 21m)   kubelet            Failed to pull image "ghcr.io/weaveworks/wego-app:0.5.0": rpc error: code = Unknown desc = Error response from daemon: manifest unknown
  Warning  Failed     20m (x4 over 21m)   kubelet            Error: ErrImagePull
  Warning  Failed     19m (x6 over 21m)   kubelet            Error: ImagePullBackOff
  Normal   BackOff    98s (x85 over 21m)  kubelet            Back-off pulling image "ghcr.io/weaveworks/wego-app:0.5.0"
```
**What is the severity of the bug**
Please add a label for this ticket indicating the severity of this bug.  

| Label | Description |
| --- | --- |
| severity/Critical | Weave GitOps is crashing or experiencing data loss, and there is no known workaround |
| severity/Major | Weave Gitops functionality is broken, there is a workaround, and the workaround requires significant effort |
| severity/Minor | Weave Gitops functionality is broken, but there is a workaround |
| severity/Low | Doesn’t affect primary flow/functionality but would be good to fix |

**Environment**
 - gitops: [v0.5.0]
 - kubernetes:  [e.g. 1.22.3]
    - [x] Minikube  [v1.24.0]
 -
**Affects versions**
0.5
**To Reproduce**
1. minikube start
2. gitops install
3. See wego-app not starting

**Expected behavior**
Wego-app starting

**Config and Logs**
If applicable, add logs to help explain your problem. _please compress the output before attaching_
- [ ] gitops config
- [ ] `gitops` cli call output (and parameters)
- [ ] gitops-controller logs
- [ ] Events from wego-* namespaces
- [ ] `kubectl cluster-info dump`
- [ ] Prometheus alerts
- [ ] Flux logs

**Screenshots**

**Additional context**

**Comment by sympatheticmoose** [comment]
Fixed by https://github.com/weaveworks/weave-gitops/pull/1181

**Comment by mtparet** [comment]
Thanks so it should fixed now ?

**Comment by sympatheticmoose** [comment]
@mtparet will be fixed in the next release which will be today/tomorrow 👍 

To document the workaround for v0.5.0 (this may be what you've already done) - I'd suggest updating the `app.yaml` in the `.weave-gitops/clusters/<cluster-name>/system with the correct image, namely `image: ghcr.io/weaveworks/wego-app:v0.5.0`

**Comment by sympatheticmoose** [comment]
@mtparet we've pushed a 0.5.0 image as well to fix 0.5.0 :) should be working now
